app/models/recipe.rb

```diff
@@ -0,0 +1,10 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods, foreign_key: 'recipe_id', dependent: :destroy
+  has_many :foods, through: :recipe_foods
+
+  def total_price
+    #   # Calculate the total price for the recipe
+    #   recipe_foods.sum { |rf| rf.quantity * rf.food.price }
```

Comment:
- [Optional] Comments render codebase messy. Please do not push to the repo. It is recommended you fix it :+1: